src/transformers/commands/serving.py

```diff
@@ -629,6 +629,14 @@ def run(self):
                 allow_methods=["*"],
                 allow_headers=["*"],
             )
+            logger.warning(
+                "WARNING: CORS allow origin is set to `*`. This is not recommended for production environments."
```

Comment:
```suggestion
                "CORS allow origin is set to `*`. This is not recommended for production environments."
```

Remove the hardcoded `WARNING:` prefix. The logger should handle the level label, so this just creates noisy duplication.